doc/release-notes/5.6-release-notes.md

```diff
@@ -0,0 +1,112 @@
+# Dataverse Software 5.6
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Anonymized Access in Support of Double Blind Review
+
+Dataverse installations can select whether or not to allow users to create anonymized private URLs and can control which specific identifying fields are anonymized. If this is enabled, users can create private URLs that do not expose identifying information about dataset depositors, allowing for double blind reviews of datasets in the Dataverse installation.
+
+### Guestbook Responses API
+
+A new API to retrieve Guestbook responses has been added. This makes it easier to retrieve the records for large guestbooks and also makes it easier to integrate with external systems.
+
+### Dataset Semantic API (Experimental)
+
+Dataset metadata can be retrieved, set, and updated using a new, flatter JSON-LD format - following the format of an OAI-ORE export (RDA-conformant Bags), allowing for easier transfer of metadata to/from other systems (i.e. without needing to know Dataverse's metadata block and field storage architecture). This new API also allows for the update of terms metadata (#5899).
+
+This development was supported by the [Research Data Alliance](https://rd-alliance.org), DANS, and Sciences PO and follows the recommendations from the [Research Data Repository Interoperability Working Group](http://dx.doi.org/10.15497/RDA00025).
```

Comment:
I put this in but I'm not sure it should be here unless it's standard. I.e. anonymized was from QDR, the Direct upload stuff was Scholars Portal, etc.

Reply:
Thanks @qqmyers - I leaned towards leaving it in because it links to the recommendations from the IWG so people can get more details. I'm OK with the provenance of contributions being noted as well, especially for these larger efforts.